_includes/segment/progress-bar.html

```diff
@@ -19,7 +19,6 @@
 {%- endcase -%}
 <div class="ui container progress bch-progress {{ page.id }}" data-value="{{ include.value }}" data-total="{{ include.total }}" id="bch-count">
 	<div class="bar {{ bar_color }}"><div class="progress"></div></div>
-	<div class="label" data-tooltip="Please help to improve this ratio by sending tweets to, or writing on the wall of the {{ list_of }} that you hope to see accept Bitcoin Cash." data-position="bottom center" data-inverted="">
-		<b>{{ include.value }}</b> out of <b>{{ include.total }}</b> {{ progress_text }}.
-	</div>
+	<div class="label" data-tooltip="Please help to improve this ratio by sending tweets to, or writing on the wall of the {{ list_of }} that you hope to see accept Bitcoin Cash." data-position="bottom center" data-inverted=""></div>
 </div>
+<p><b>{{ include.value }}</b> out of <b>{{ include.total }}</b> {{ progress_text }}.</p>
```

Comment:
@t0p0lin0  it appears this gives us a huge amount of spacing and the text looks more for the progress bar below it when on mobile. Perhaps we bring the paragrpah opening tag to before the progress bar starts so its better wrapped?

Reply:
Can you put this in a separate branch I can pull?

Reply:
actually, I gave it a try and it looked worse than before. But something should be done to help the separation on mobile, very confusing the text that matches to the bar